json/es.json

```diff
@@ -1,6 +1,6 @@
 {
     "A fresh verification link has been sent to your email address.": "Se ha enviado un nuevo enlace de verificación a su correo electrónico.",
-    "A new verification link has been sent to the email address you provided during registration.": "A new verification link has been sent to the email address you provided during registration.",
+    "A new verification link has been sent to the email address you provided during registration.": "Se ha enviado un nuevo enlace de verificación a la dirección de correo electrónico que proporcionó durante el registro..",
```

Comment:
Thanks, but this line already was translated in the PR (https://github.com/Laravel-Lang/lang/pull/1341/files). We are waiting a review and a possible merge. This line besides contains an innecessary double dot at end.